cmd/chainbridge/flags.go

```diff
@@ -52,6 +48,10 @@ var (
 		Name:  "ethereum",
 		Usage: "Import an existing ethereum keystore",
 	}
+	PrivateKeyFlag = cli.StringFlag{
+		Name:  "privateKey",
+		Usage: "Hex string private key used to generate a keypair.",
```

Comment:
Please document it must be `0x` prefixed

Reply:
We have some code that checks if its `0x` prefixed and removes it if it is, otherwise leaves it alone. I would assume this functionality allows for both `0x` prefix and non-prefix keys?
https://github.com/ChainSafe/ChainBridge/pull/319/files/054043bb963be2543abbfc0452287ab77301a94a#diff-862b5524ef0f947da0f6611cafb4d210R99-R102